plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile

```diff
@@ -1311,6 +1269,250 @@ bc. myBool = active(StateA)
+
+
+
+h3(#sclang_examples). Examples
+
+Parent-first versus child-first execution are best explained by a couple of examples.
+
+
+h4. Simple child-first/parent-first examples
+
+==<div class="example">==
+
+p(#sclang_example_1a). Example 1a:
+
+Consider the statechart in the "figure below":#fig_child-first_parent-first_example_1a-1. Composite state *A* and its substate *B* are active, and the <code>@ChildFirstExecution</code> annotation has been specified in the definition section.
+
+p(#fig_child-first_parent-first_example_1a-1). 
+
+!images/docu_childfirst_010_example_01a-1.png(Child-first, example 1a, before transition)!
+
+p=. Child-first, example 1a, before transition
+
+When event _e_ occurs, the state machine considers child state *B* first, finds a matching transition, and thus immediately proceeds to state *D*, see the "figure below":#fig_child-first_parent-first_example_1a-2.
+
+p(#fig_child-first_parent-first_example_1a-2). 
+
+!images/docu_childfirst_010_example_01a-2.png(Child-first, example 1a, after transition)!
+
+p=. Child-first, example 1a, after transition
+
+==</div>==
+
+
+
+==<div class="example">==
+
+p(#sclang_example_1b). Example 1b:
+
+Consider the statechart in the "figure below":#fig_child-first_parent-first_example_1b-1. Composite state *A* and its substate *B* are active, and the <code>@ParentFirstExecution</code> annotation has been specified in the definition section.
+
+p(#fig_child-first_parent-first_example_1b-1). 
+
+!images/docu_childfirst_010_example_01b-1.png(Parent-first, example 1b, before transition)!
+
+p=. Parent-first, example 1b, before transition
+
+When event _e_ occurs, the state machine considers parent state *A* first, finds a matching transition, and thus immediately proceeds to state *C*, see the "figure below":#fig_child-first_parent-first_example_1b-2.
+
+p(#fig_child-first_parent-first_example_1b-2). 
+
+!images/docu_childfirst_010_example_01b-2.png(Parent-first, example 1b, after transition)!
+
+p=. Parent-first, example 1b, after transition
+
+==</div>==
+
+
+h4. Child-first/parent-first and local reactions
+
+The following examples are somewhat more complex. The statechart used in the examples defines the integer variables _m_, _n_, _o_, _p_, _q_, and _r_. Initially, they are all set to 0. Local reactions in states *A* and *B* set these variables to non-zero values on certain conditions:
+* when entering a state,
+* when leaving a state, and
+* when receiving event _e_.
+
+This is useful to understand which actions are performed (or not performed) and in which order.
+
+The local reactions in composite state *A* are as follows:
```

Comment:
entry and exit actions play a specific role within state transitions and are no local reactions even though they are defined in the same location. Please also be aware that you can combine the triggers like ' e, entry, exit / doSomething' . In this case a local reaction as well as entry and exit actions are defined. (It is not neccessary to explain this here....)
  
You could write something like 'The reactions defined in composite state "A" ...."

Reply:
Ah, I wasnt aware that entry/exit reactions don't count as "local" reactions! On the other hand, does such a differentiation really matter to the user (or to anyone else)? Wouldn't it be easier for everyone to define all reactions that are "inside the box" as being "local reactions"?